plugins/Installation/lang/en.json

```diff
@@ -137,6 +137,8 @@
         "SystemCheckPhpSetting": "To prevent some critical issue, you must set the following in your php.ini file: %s",
         "SystemCheckUpdateHttps": "Update over HTTPS",
         "SystemCheckUpdateHttpsNotSupported": "Matomo cannot use HTTPS to update, it will fall back to the insecure HTTP update. Check that CURL or allow_url_fopen is supported and that the openssl PHP extension is installed: https:\/\/matomo.org\/faq\/troubleshooting\/faq_177\/.",
+        "SystemCheckProcessorFormat" : "64-bit PHP Binary",
+        "SystemCheckProcessorFormat32": "32-bit<p> We highly recommend upgrading to a 64-bit PHP binary to prevent various issues that may cause inconvenience in the future.</p>",
```

Comment:
The translation key should be renamed, so they either describe what they contain or where they are used. Those keys are visible to the translators and should (if possible) kind of help them to understand the context.